packages/table/src/utils/messages.ts

```diff
@@ -0,0 +1,56 @@
+import { PropertyFilterProps } from '@awsui/components-react';
+
+export type PropertyFilterMessages = PropertyFilterProps.I18nStrings;
+export type TableCellMessages = {
+  loading: string;
+};
+
+export type TableMessages = {
+  propertyFilter: PropertyFilterMessages;
+  tableCell: TableCellMessages;
+};
+
+export type PartialTableMessages = Partial<{
```

Comment:
instead, use `RecursivePartial`, and we can skip ever declaring this, and just directly use `RecursivePartial<TableMessages>` as the typescript type on the prop

Reply:
Fixed it.